Library/Homebrew/extend/os/mac/diagnostic.rb

```diff
@@ -90,6 +91,18 @@ def check_clt_up_to_date
         EOS
       end
 
+      def check_xcode_8_without_clt_on_el_capitan
+        return unless MacOS::Xcode.without_clt?
+        # Scope this to Xcode 8 on El Cap for now
+        return unless MacOS.version == :el_capitan && MacOS::Xcode.version >= "8"
+
+        <<-EOS.undent
+          You have Xcode 8 installed without the CLT; this causes certain builds to fail.
+          Please install the CLT via:
+            xcode-select --install
```

Comment:
Maybe a good idea to recommend `sudo` here as `xcode-select` will only abort unless you give it elevated privs.

If we're using a vanilla non-sudo recommendation everywhere else though feel free to ignore me. No point introducing an inconsistency.